README.md

```diff
@@ -26,14 +26,76 @@ would be a nice experience.
 Of course, in order to understand the Dockerfile, one might need some introduction.
 So we tried to document each line in the Dockfile for you.
 
+## Getting Started
 
+### Prerequisites
 
+In order to use this repo, you should install:
 
-# Build
+* Docker : For building and run the docker image
+* git: For cloning the repo
+* git-lfs : This is needed because ACS repo keeps larger files that are served with this system. Hopefully, we will remove this in the future.
 
-    docker build . --tag=acs:2020.4
 
-# Run
+Once you have the aforementioned prerequisites, clone the repo like this:
 
-    docker run -dP --name=acs acs:2020.4
+```
+git clone --recursive https://github.com/dneise/acs_test acs_docker
+```
 
+NOTE: This command should **also** clone the ACS repo. If this not happen, once the ACS Docker repo is cloned, checkout the develop branch:
+
+```
+git checkout develop
+```
+
+And update the submodule:
+
```

Comment:
let's add a "this might take a while" here ?

Reply:
Agree

Reply:
done